.github/workflows/cicd.yml

```diff
@@ -253,17 +259,9 @@ jobs:
         with:
           python-version: "3.9"
 
-      - name: "Combine coverage"
-        run: |
-          set -eux
-          pip install coverage[toml];
-          coverage combine;
-          coverage xml;
-        env:
-          COVERAGE_FILE: .tox/coverage
-
```

Comment:
This is unrelated to the `pypy` change, right?

Reply:
Thanks for the review and the merge.

Yes. Not a hard requirements.

But this step was failing under pypy as there was no coverage.

Rather than add an `if` to this step, I found that tox already has combine and pip install logic, so I moved the XML part to tox.